code/game/machinery/autolathe.dm

```diff
@@ -116,21 +116,25 @@
 		if(do_after(user, 14.4, target = src))
 			for(var/B in D.blueprints)
 				if(B)
+					var/datum/design/design = B
+					if(!design.build_type & AUTOLATHE)
+						continue

```

Comment:
This section is not needed. This behaviour is handled in code/modules/research/rdconsole.dm (even though it's not very pretty) https://github.com/tgstation/tgstation/pull/45118/files#diff-6ed7ac497af1a014f7f9c82cf2d9bb9eR1018

Should fix Tlals problem in #45197

Reply:
fixed, thanks.